tests/integration-tests/clusters_factory.py

```diff
@@ -63,20 +64,39 @@ def update(self, force=True):
             logging.error(error)
             raise Exception(error)
         logging.info("Cluster {0} updated successfully".format(self.name))
+        # Only update config file attribute if update is successful
+        self.config_file = config_file
+        with open(self.config_file) as conf_file:
+            self.config = yaml.load(conf_file, Loader=yaml.SafeLoader)
```

Comment:
the alternative to `yaml.load(... Loader=yaml.SafeLoader)` should be simply `yaml.safe_load`

Reply:
There are many appearances, I uniformed them all